src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,198 @@
+package duke.task;
+
+import duke.command.Parser;
+import java.util.ArrayList;
+import java.util.Calendar;
+
+/**
+ * Represents an duke.task.Event object that has a description, start-time and end-time to depict a duration.
+ */
+public class Event extends Task {
+    private Calendar startTime;
+    private Calendar endTime;
+
+    /**
+     * Creates duke.task.Event object taking info from Parser's arrayList.
+     * @param parser
+     */
+    public Event(Parser parser) {
+        super(parser.getList().get(0));
+        startTime = Calendar.getInstance();
+        endTime = Calendar.getInstance();
+        ArrayList<String> inputArray = parser.getList();
+        startTime.set(Integer.parseInt(inputArray.get(3)),
+                (Integer.parseInt(inputArray.get(2)) - 1),
+                Integer.parseInt(inputArray.get(1)),
+                Integer.parseInt(inputArray.get(4)),
+                Integer.parseInt(inputArray.get(5)));
+        endTime.set(Integer.parseInt(inputArray.get(3)),
+                (Integer.parseInt(inputArray.get(2)) - 1),
+                Integer.parseInt(inputArray.get(1)),
+                Integer.parseInt(inputArray.get(6)),
+                Integer.parseInt(inputArray.get(7)));
+    }
+
+    /**
+     * Creates duke.task.Event object using description, start, and end-time.
+     * @param des description of Task.
+     * @param startTime startTime of duke.task.Event.
+     * @param endTime endTime of duke.task.Event.
+     */
+    public Event(String des, Calendar startTime, Calendar endTime) {
+        super(des);
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+
+    /**
+     * Return startTime of event.
+     * @return startTime.
+     */
+    public Calendar getStartTime() {
+        return startTime;
+    }
+
+    /**
+     * Return endTime of event.
+     * @return endTime.
+     */
+    public Calendar getEndTime() {
+        return endTime;
+    }
+
+    /**
+     * Returns a string representing startTime in a particular user-friendly format.
+     * @return string representing the startTime in a printing friendly format.
+     */
+    public String printStartTime() {
```

Comment:
Would it be possible to move this to `Task` class so it does not have to be repeated in `Event` and `Deadline` classes?

Reply:
I did not move it to Task because ToDo tasks do not have time so having a printTime method for it's superclass might not make lot of sense. 
Would that matter?